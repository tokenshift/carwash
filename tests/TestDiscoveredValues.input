PASSWORD=super_secret
PASSWORD = also_secret
PASSWORD='testpass'
PASSWORD="whatever"
PASSWORD my_voice_is_my
PASSWORD "sensitive"
PASSWORD 'alsosensitive'
PASSWORD: wordpass
PASSWORD: 'w0rdp4ss'
PASSWORD: "w0rdp3rf3ct"
PASSWORD => ruby_code
PASSWORD => "No, really!"
PASSWORD => 'Ok, fine.'
"PASSWORD" => "I'll quote the key too."
"PASSWORD" => "But you can't handle \"this\"!"
'PASSWORD' => 'Or \'this\'.'
This is a log line: { :some_password => "testing1", :another_password => "testing2", :something_else => "testing3" }
password=methusaleh
PaSsWoRd=helasuthem
super_secret SUPER_secret methusaleh helasuthem
ENV password scherzo_in_e_minor
ENV password=annella_perfume
But you can't handle "this"!
Or 'this'.
