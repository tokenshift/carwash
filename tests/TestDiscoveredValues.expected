PASSWORD=********
PASSWORD = ********
PASSWORD='********'
PASSWORD="********"
PASSWORD ********
PASSWORD "********"
PASSWORD '********'
PASSWORD: ********
PASSWORD: '********'
PASSWORD: "********"
PASSWORD => ********
PASSWORD => "********"
PASSWORD => '********'
"PASSWORD" => "********"
"PASSWORD" => "********"
'PASSWORD' => '********'
This is a log line: { :some_password => "********", :another_password => "********", :something_else   => "********" }
password=********
PaSsWoRd=********
******** ******** ******** ********
ENV password ********
ENV password=********
********
********

